teaspoons-cli/teaspoons/config.py

```diff
@@ -0,0 +1,24 @@
+# config.py
+
+from pathlib import Path
+from oauth2_cli_auth import OAuth2ClientInfo
+from dotenv import dotenv_values
+
+
+class CliConfig:
+    """A class to hold configuration information for the CLI"""
+
+    def __init__(self):
+        self.config = dotenv_values("../.env")
+
+        self.client_info = OAuth2ClientInfo.from_oidc_endpoint(
+            self.config["OAUTH_OPENID_CONFIGURATION_URI"],
+            client_id=self.config["OAUTH_CLIENT_ID"],
+            scopes=[f"openid+email+profile+{self.config['OAUTH_CLIENT_ID']}"]
+        )
+
+        # Figure out the path to the token file...there must be some way to abstract out local storage
```

Comment:
what does this comment mean about abstracting local storage?  like not needing to save the "path" in the config file or something else?

Reply:
honestly not sure, Muscles left this comment. I think he was hoping there'd be a more elegant way to do the following if-else statement, but what he has makes sense to me. should i remove this comment (or the second half of it)?

Reply:
im trying to think when we would ever set the `LOCAL_STORAGE_PATH` to start with a `/` 🤔.  we can probably take out the if part of the if/else and add it back in if we ever feel like its necessary?